lms/djangoapps/teams/static/teams/js/views/teams_tab.js

```diff
@@ -208,10 +208,17 @@
                         courseID = this.courseID;
                     self.getTopic(topicID).done(function(topic) {
                         self.getTeam(teamID).done(function(team) {
-                            var view = new TeamProfileView({
-                                courseID: courseID,
-                                model: team
-                            });
+                            var readOnly = !(
+                                self.$el.data('privileged') ||
```

Comment:
I recommend breaking this out into an ``isReadOnly`` helper method, and then have some unit tests for the different combinations.